config/routes.rb

```diff
@@ -12,4 +12,7 @@
              }
   get '/books', to: 'books#index'
   get '/books/:id', to: 'books#show'
+
+  post '/api/v1/rents', to: 'rents#create'
```

Comment:
Why are you adding now the /api/v1 path but you didn't for the book routes? Try to maintain the same path for all routes.